docs/basics/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Basic Tutorial
+permalink: /basics/
+has_children: true
+nav_order: 2
+has_toc: false
+---
+
+# Basic Tutorial
+
+The basic tutorial is a work in progress. It will be directed towards beginners once its ready.
```

Comment:
`its` ➡️ `it's`

Reply:
Thanks, it's fixed! And no worries about being That Guy: there are whole job profiles who do nothing else than fix typos, it's important 😄